gef.py

```diff
@@ -8442,17 +8415,17 @@ def print_arguments_from_symbol(self, function_name: str, symbol) -> None:
         for i, f in enumerate(symbol.type.fields()):
             _value = gef.arch.get_ith_parameter(i, in_func=False)[1]
             _value = RIGHT_ARROW.join(dereference_from(_value))
-            _name = f.name or "var_{}".format(i)
+            _name = f.name or f"var_{i}"
             _type = f.type.name or self.size2type[f.type.sizeof]
-            args.append("{} {} = {}".format(_type, _name, _value))
+            args.append(f"{_type} {_name} = {_value}")
 
         self.context_title("arguments")
 
         if not args:
-            gef_print("{} (<void>)".format(function_name))
+            gef_print(f"{function_name} (<void>)")
             return
 
-        gef_print("{} (".format(function_name))
+        gef_print(f"{function_name} (")
         gef_print("   " + ",\n   ".join(args))
         gef_print(")")
```

Comment:
```suggestion
        gef_print(f"{function_name} ("
                  "\n  {',\n   '.join(args)}")
                  "\n)")
```

Even if not, we can put an f string in here.

Reply:
yeah, this won't work as I just said above. f-string don't allow backslashes in their "parts"